pack/apk.mk

```diff
@@ -39,11 +40,13 @@ package: $(BUILDDIR)/$(TARBALL) \
 	@echo "Building APK packages"
 	@echo "-------------------------------------------------------------------"
 	sudo apk update
+	sudo chown -R ${USER}:${USER} /home/${USER}/.abuild/
```

Comment:
The assumption that the primary group is named the same as the user may be wrong. For example, my system:

```shell
$ id -nu
alex
$ id -ng
users
```

(IDs are different too.)

Reply:
OK, I replace it with changing owner only.

Reply:
Irrelevant now.